demisto_sdk/commands/init/initiator.py

```diff
@@ -378,6 +397,25 @@ def get_valid_user_input(options_list: List[str], option_message: str) -> str:
             except ValueError:
                 user_input = input("\nThe option must be number, please enter valid choice: ")
 
+    def ignore_secrets_automatically(self):
+        pack_dir = get_pack_name(self.full_output_path)
+        files_and_directories = glob.glob(f'{self.full_output_path}/**/*', recursive=True)
+
+        sv = SecretsValidator(white_list_path='./Tests/secrets_white_list.json')
+        # remove directories and irrelevant files
+        files = [file for file in files_and_directories if os.path.isfile(file) and sv.is_text_file(file)]
+        # The search_potential_secrets method returns a nested dict with values of type list. The values are the secrets
+        # {'a': {'b': ['secret1', 'secret2'], 'e': ['secret1']}, 'g': ['secret3']}
+        all_secrets = sv.search_potential_secrets(files)
+        secrets: set = set()
```

Comment:
Please improve the naming here. Its not clear what is the difference between "secrets" and "all secrets"